### PR TITLE
Adding Catalogs design artifacts

### PR DESCRIPTION
**Pull request by serenamarie125**

### CODE REVIEW

**Comment by serenamarie125** [comment]
@beanh66 Can you take a look at this ?

**Comment by beanh66** [comment]
@serenamarie125 Yes, this looks good to me! Just added one comment inline

File: UX/ui-service/Catalogs/design.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+- The Create action menu is available in the Configuration button group
+- Create should always be enabled
+
+#### Implementation Details
+- Note that this is not yet a part of PatternFly - we may want to look into getting support
+
+### Create Catalog Flow
+![Create Catalog Flow - Screen 1](img/Catalogs-CreateFlow-01.png)
+- The create flow is accomplished in a page, not in a modal
+- Include a breadcrumb on the page
```
Comment (beanh66) [inline UX/ui-service/Catalogs/design.md]: @serenamarie125 The breadcrumbs in the Create Catalog section will not actually have the Catalog Name yet, so it should be noted that this breadcrumb should read "Designer >> Catalogs >> New Catalog" until it is actually saved.

**Comment by cshinn** [comment]
@serenamarie125 Added a small comment, but looks good overall!

**Pull request merged**